GitHubRepositories.ps1

```diff
@@ -1607,3 +1607,558 @@ function Move-GitHubRepositoryOwnership
 
     return Invoke-GHRestMethod @params
 }
+
+Filter Test-GitHubRepositoryVulnerabilityAlert
+{
+ <#
+    .SYNOPSIS
+        Retrieves the status of vulnerability alerts for a repository on GitHub.
+
+    .DESCRIPTION
+        Retrieves the status of vulnerability alerts for a repository on GitHub.
+
+        The Git repo for this module can be found here: http://aka.ms/PowerShellForGitHub
+
+    .PARAMETER OwnerName
+        Owner of the repository.
+        If not supplied here, the DefaultOwnerName configuration property value will be used.
+
+    .PARAMETER RepositoryName
+        Name of the repository.
+        If not supplied here, the DefaultRepositoryName configuration property value will be used.
+
+    .PARAMETER Uri
+        Uri for the repository.
+        The OwnerName and RepositoryName will be extracted from here instead of needing to provide
+        them individually.
+
+    .PARAMETER AccessToken
+        If provided, this will be used as the AccessToken for authentication with the
+        REST Api.  Otherwise, will attempt to use the configured value or will run unauthenticated.
+
+    .PARAMETER NoStatus
+        If this switch is specified, long-running commands will run on the main thread
+        with no commandline status update.  When not specified, those commands run in
+        the background, enabling the command prompt to provide status information.
+        If not supplied here, the DefaultNoStatus configuration property value will be used.
+
+    .INPUTS
+        GitHub.Repository
```

Comment:
This isn't limited to a `GitHub.Repository`.  Any object that has a `RepositoryUrl` property will work here.  Review the `.INPUTS` for `Get-GitHubRepository` for the current complete list.  For example, I can pipe an Issue in here if I want to see the test if security vulnerability alerts are enabled for the repo that the issue is part of.  Here and below.

Reply:
Done